db/migrations/20221031120849_orders.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE orders (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    check_in DATETIME NOT NULL,
+    check_out DATETIME NOT NULL,
```

Comment:
네이밍을 명확하게 해주는 것이 좋아보입니다.
```suggestion
    check_in_date DATETIME NOT NULL,
    check_out_date DATETIME NOT NULL,
```